src/test/java/org/jenkinsci/plugins/saml/SamlSecurityRealmTest.java

```diff
@@ -78,6 +90,20 @@ public void testReadSimpleConfiguration() {
         assertEquals("urn:mace:dir:attribute-def:mail", samlSecurityRealm.getEmailAttributeName());
         assertEquals("urn:mace:dir:attribute-def:uid", samlSecurityRealm.getUsernameAttributeName());
         assertEquals(true, samlSecurityRealm.getIdpMetadata().startsWith("<?xml version"));
+        assertEquals(SAML2_REDIRECT_BINDING_URI, samlSecurityRealm.getBinding());
+    }
+
+    @LocalData
+    @Test
+    public void testReadSimpleConfigurationHTTPPost() {
```

Comment:
Is this test checking backward compatibility? IOW does the `LocalData` contain _old_ config files?

Reply:
testReadSimpleConfiguration uses an old config file without the field